src/gui/main.cpp

```diff
@@ -94,6 +94,7 @@ int main(int argc, char **argv)
 
     QCoreApplication::setAttribute(Qt::AA_UseHighDpiPixmaps, true);
     QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling, true);
+	QApplication::setHighDpiScaleFactorRoundingPolicy(Qt::HighDpiScaleFactorRoundingPolicy::PassThrough);
```

Comment:
Nitpick -- could you fix indentation? Thanks!

Reply:
Ok, should be all space-y now!